python.d/springboot.chart.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+# Description: tomcat netdata python.d module
+# Author: Wing924
+
+import json
+from bases.FrameworkServices.UrlService import UrlService
+
+# default module values (can be overridden per job in `config`)
+# update_every = 2
+priority = 60000
+retries = 60
+
+
+DEFAULT_ORDER = ['response_code', 'threads', 'gc_time', 'gc_ope', 'heap']
+
+DEFAULT_CHARTS = {
+    'response_code': {
+        'options': [None, "Response Codes", "requests/s", "response", "springboot.response_code", "stacked"],
+        'lines': [
+            ["resp.other",  'Other', 'incremental'],
+            ["resp.1xx",    '1xx',   'incremental'],
+            ["resp.2xx",    '2xx',   'incremental'],
+            ["resp.3xx",    '3xx',   'incremental'],
+            ["resp.4xx",    '4xx',   'incremental'],
+            ["resp.5xx",    '5xx',   'incremental'],
+        ]},
+    'threads': {
+        'options': [None, "Threads", "current threads", "threads", "springboot.threads", "area"],
```

Comment:
stacked?

Reply:
It should be area because daemon is a kind of thread.